frontend/src/FiltersTablePage/LeadersTable.tsx

```diff
@@ -0,0 +1,102 @@
+import { TableCell, TableRow, TableHead, Table, List, ListItem, TableBody, Card, TableContainer } from '@mui/material'
+import { Link } from 'react-router-dom'
+import CheckIcon from '@mui/icons-material/Check'
+import { leaderData, _getLeadersData } from './_testFunctions'
+import { Box, styled } from '@mui/system'
+
+const StyledTableCell = styled(TableCell)({
+    fontSize: '11px',
+    size: 'small',
+    padding: 6,
+    borderRightWidth: 1,
+    borderRightColor: "lightgray",
+    borderRightStyle: "solid",
+    margin: 3
+})
+
+function Head() {
+    const columns = ['id', 'Name', 'City', 'Category', 'Projects', 'Interviewer', 'Date',
```

Comment:
This page is not dynamically generated, is it? These hard-coded columns don't look like that.